substrate/runtime/src/lib.rs

```diff
@@ -292,6 +319,36 @@ impl assets::Config for Runtime {
   type BenchmarkHelper = SeraiAssetBenchmarkHelper;
 }
 
+pub type PoolAssetsInstance = assets::Instance1;
+impl assets::Config<PoolAssetsInstance> for Runtime {
+  type RuntimeEvent = RuntimeEvent;
+  type Balance = SubstrateAmount;
+  type RemoveItemsLimit = ConstU32<1000>; // 0?
```

Comment:
Add a TODO if this is an active question?

Reply:
yes it was a question lol. Same type `RemoveItemsLimit` also exist in assets pallet config. So should they be same ?

Reply:
Making the same is fine. They may have different costs of removal, so the actual limit on execution may be distinct, but I'm not concerned about it at this time.